cli/dcoscli/node/main.py

```diff
@@ -473,7 +473,7 @@ def _list(json_):
     if json_:
         emitter.publish(slaves)
     else:
```

Comment:
Well we can't really check that here since there's no schema and some rows could have a field while others don't, but that's a good idea to have better handling of when we can't get a value.

See e072b4becf1eed4ef031c08eb90d43c70918f56c.

```
$ dcos node --field='Ethos Role:attributes.ethos_role' --field='disk used:used_resources.disk' --field='foobar'
  HOSTNAME         IP                          ID                    ETHOS ROLE  DISK USED  FOOBAR
10.10.12.174  10.10.12.174  74df965e-a46e-46a9-8d2a-639db0af0e25-S0    worker      4096.0    ???
10.10.12.183  10.10.12.183  74df965e-a46e-46a9-8d2a-639db0af0e25-S4    proxy       4096.0    ???
10.10.12.29   10.10.12.29   74df965e-a46e-46a9-8d2a-639db0af0e25-S2   control      5096.0    ???
10.10.3.201   10.10.3.201   74df965e-a46e-46a9-8d2a-639db0af0e25-S3    public       0.0      ???
```